src/pages/public_registration/LogPanel.tsx

```diff
@@ -109,31 +115,70 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
-            {organizationQuery.isLoading || userQuery.isLoading ? (
-              <Skeleton sx={{ width: '4rem' }} />
-            ) : (
-              <Typography>
-                {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
-                {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
-              </Typography>
-            )}
-          </Box>
-          <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          <Typography>{t('common.created')}</Typography>
+          <Tooltip title={new Date(registration.createdDate).toLocaleTimeString()} enterDelay={tooltipDelay}>
+            <Typography>{new Date(registration.createdDate).toLocaleDateString()}</Typography>
+          </Tooltip>
+          {organizationQuery.isLoading || userQuery.isLoading ? (
+            <Skeleton sx={{ width: '4rem' }} />
+          ) : (
+            <Typography>
+              {organizationQuery.data ? organizationQuery.data?.acronym : t('common.unknown')}
+              {userQuery.data ? `, ${getFullName(userQuery.data.givenName, userQuery.data.familyName)}` : ''}
+            </Typography>
+          )}
         </StyledStatusMessageBox>
       )}
       {logs
         .sort((a, b) => new Date(a.modifiedDate).getTime() - new Date(b.modifiedDate).getTime())
         .map((logItem, index) => {
           const bgColor = logItem.type === 'PublishingRequest' ? 'publishingRequest.main' : 'doiRequest.main';
+          const modifiedDate = new Date(logItem.modifiedDate);
           return (
             <StyledStatusMessageBox key={index} sx={{ bgcolor: `${bgColor}` }}>
               <Typography>{logItem.description}</Typography>
-              <Typography>{new Date(logItem.modifiedDate).toLocaleDateString()}</Typography>
+              <Tooltip title={modifiedDate.toLocaleTimeString()} enterDelay={tooltipDelay}>
+                <Typography>{modifiedDate.toLocaleDateString()}</Typography>
+              </Tooltip>
+              {logItem.fileNames && logItem.fileNames.length > 0 && (
+                <Box
+                  component="ul"
+                  sx={{
+                    gridColumn: '1/3',
+                    m: '0',
+                    p: 'inherit',
+                    color: 'black',
+                  }}>
+                  {logItem.fileNames.map((desc, index) => (
+                    <li key={index}>
+                      <Tooltip title={desc === t('common.deleted') ? '' : desc} enterDelay={tooltipDelay}>
+                        <Typography
+                          sx={{
+                            whiteSpace: 'nowrap',
+                            overflow: 'hidden',
+                            textOverflow: 'ellipsis',
+                            fontStyle: desc === t('common.deleted') ? 'italic' : 'normal',
+                            width: 'fit-content',
```

Comment:
`italic` og `fit-content` spiller visst ikke helt på lag, da noe av det kursive vil komme utenfor feltet og dermed kuttes. merker ikke så godt på norsk, men på engelsk blir det "Deletea" når oversettelsene er på plass 😅 
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/bdf2cd9c-7a20-4472-ba9a-a250d17473b4)
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/7df543eb-8eea-45a2-9ed0-72096abf0b8f)

Vet ikke helt hva den beste fiksen er her, men tipper det er å unngå `fit-content` på noe vis. Ikke noen deal-breaker for meg nå, men kan være verdt å sjekke ut om det er en enkel løsning